cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/inmemory/InMemoryStreamAdmin.java

```diff
@@ -153,6 +154,14 @@ public ViewSpecification getView(Id.Stream.View viewId) throws Exception {
   }
 
   @Override
+  public boolean viewExists(Id.Stream.View viewId) throws Exception {
```

Comment:
The exists calls throws Exception if it fails to read from store cant user a lower exception from hierarchy in throws